infrastructure/smart-contracts/hardhat.config.cjs

```diff
@@ -54,6 +54,10 @@ const config = {
 		  // blockNumber: 184085729,
      // },
       chainId: 1,
+      accounts:{
+        accounts,
+        accountsBalance: "100000000000000000000000000000" 
```

Comment:
This needs to go up where `const accounts` is defined, it does not work in here and hardhat will default to 10K ETH.

Reply:
I'll just remove it, if it's not being used, the tests run fine as is.